context-free.md

```diff
@@ -1,22 +1,25 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+People who want to create digital art. 
 
 ## What is easy to do in this language? Why is it easy?
-
+It is easy to create images recursively with the looping syntax - things like spiral, snowflakes and trees. It's also easy to implement randomness by creating different versions of a shape and assigning weights to them. 
```

Comment:
Good point about the randomness aspect - this is something that I missed in my description!